src/InAppBrowser.android.ts

```diff
@@ -3,13 +3,13 @@ import Bundle = android.os.Bundle;
 import TextUtils = android.text.TextUtils;
 import Intent = android.content.Intent;
 import Context = android.content.Context;
-import Color = android.graphics.Color;
+// import Color = android.graphics.Color;
```

Comment:
Please don't comment code, remove this instead

Reply:
Done